locales/en-US/tools.ftl

```diff
@@ -60,6 +60,10 @@ tools-install-page-title = Install Rust
 
 install-using-rustup-heading = Using rustup (Recommended)
 
+install-rustup32-button = Download <nobr>rustup-init.exe</nobr> <nobr>(32-bit)</nobr>
```

Comment:
`nobr` is [non-standard](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr). 
```suggestion
install-rustup32-button = Download <span class="nowrap">rustup-init.exe</span> <span class="nowrap">(32-bit)</span>
```